provider/schema/resource.go

```diff
@@ -52,7 +52,7 @@ func (r *Resource) Keys() []string {
 	for _, primKey := range tablePrimKeys {
 		data := r.Get(primKey)
 		if data != nil {
-			results = append(results, fmt.Sprintf("%v", data))
+			results = append(results, fmt.Sprintf("%s", data))
```

Comment:
does print `%s` with `interface{}` type work? are we sure? do all our data have stringer, or?